sql/utils.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+
+def get_sql_for_connection(schema_editor, direction: str) -> str:
```

Comment:
I appreciate what you're trying to do here but let's move it to the migrations folder. This sql module should not exist.

Just move this function to the migration itself, and have the sql files in `djstripe/migrations/sql/0013_postgres_forwards.sql` etc.

Reply:
@jleclanche Changes made. Please re-review.